### ISSUE TITLE
[BUG]: Incorrect default actor initialization in BLOOMActor class of ChatGPT.

### ISSUE THREAD

**Issue by snowyday** [hpcaitech/ColossalAI#3010]
### 🐛 Describe the bug

Hello, 
I wanted to report a bug with the current implementation of the [inference.py](https://github.com/hpcaitech/ColossalAI/blob/main/applications/ChatGPT/examples/inference.py#L29).
Specifically, when using the bloom model, the actor is not being initialized correctly. 
At the moment, the actor is being initialized using the following command:

```
python inference.py --pretrain <your actor model path> --model bloom
```

https://github.com/hpcaitech/ColossalAI/blob/82503a96f20c2e61016af78445f9d1697ac07dd5/applications/ChatGPT/examples/inference.py#L14

However, the correct way to initialize the actor when using bloom is to specify the corresponding tokenizer configuration, like this:

```
actor = BLOOMActor('bigscience/bloom-560m').to(torch.cuda.current_device())
```

https://github.com/hpcaitech/ColossalAI/blob/82503a96f20c2e61016af78445f9d1697ac07dd5/applications/ChatGPT/examples/inference.py#L29

This will load the specified model. I suggest updating/modifying the BLOOMActor class to initialize the actor with the correct default model.

### Environment

current main branch

**Comment by FrankLeeeee** [comment]
Hi @Fazziekey  and @ht-zhou  , can you help to answer this question?

**Comment by ht-zhou** [comment]
Thank you for your feedback, this problem has been solved in #2988 .

**Comment by ht-zhou** [comment]
I'm going to close this issue, and feel free to reopen this if you have any other questions.
